src/Compilers/Core/Portable/CodeAnalysisResources.resx

```diff
@@ -627,4 +627,10 @@
   <data name="InvalidOperationBlockForAnalysisContext" xml:space="preserve">
     <value>Given operation block does not belong to the current analysis context.</value>
   </data>
+  <data name="IsSymbolAccessibleBadWithin" xml:space="preserve">
```

Comment:
Should we instead offer two overloads of this method, one with `INamedTypeSymbol` and one with `IAssemblySymbol` (similar to some other semantic model APIs)?

Reply:
We considered that when the API was reviewed. No.

---
In reply to: [202469043](https://github.com/dotnet/roslyn/pull/27312#discussion_r202469043) [](ancestors = 202469043)